scripts/jenkins/qa_openstack.sh

```diff
@@ -168,8 +168,10 @@ $zypper --gpg-auto-import-keys -n ref
 # deinstall some leftover crap from the cleanvm
 $zypper -n rm --force 'python-cheetah < 2.4'
 
-# deinstall cloud-init and dependencies
+# deinstall cloud-init and dependencies (but keep cloud-final systemd unit around)
+cp /usr/lib/systemd/system/cloud-final.service /tmp
 $zypper -n rm --force -u cloud-init
+cp /tmp/cloud-final.service /usr/lib/systemd/system/cloud-final.service
```

Comment:
> deamonize the cleanvm run, so that it will not be killed by systemd (maybe run in detached 
> screen, like install-suse-cloud)

I wouldn't trust that, since systemd has started to show interest in these kinds of processes. I'd rather tell systemd that I want this process to run using systemd mechanisms - anything else may fall victim to future behavioural changes in systemd.

> or fix/update dependecies of cloud-init (so that it does not need to be uninstalled)

I like that one better, but I'd rather make this a follow-up.